### PR TITLE
fixed the code

### PR DESCRIPTION
**Pull request by Zodwa679**

### CODE REVIEW

File: Task0.8.js [hunk 1]
```diff
@@ -10,11 +10,11 @@ function  convertNumberTime(number){
...
     }
     else {
-        return minutes > 0 ? hourText + muniteText : hourText; 
+        console.log(minutes > 0 ? hourText + muniteText : hourText); 
     }
 }
     
-console.log(convertNumberTime(133));
\ No newline at end of file
+console.log(convertNumberTime(133));
```
Comment (nololie) [inline Task0.8.js]: Remove the console log and call the function only. It will do the printing.

**Comment by nololie** [comment]
All looks and works perfectly fine. Well done!

**Comment by nololie** [comment]
May you please grant me write access to your project so that I can approve.

**Comment by Zodwa679** [comment]
Hi Lawrence

How do I grant you the access I thought when my project is public you could
have the access?

**Pull request merged**